beetmoverscript/script.py

```diff
@@ -176,20 +176,17 @@
 
 async def push_to_maven(context):
     """Push artifacts to locations expected by maven clients (like mvn or gradle)"""
-    artifacts_to_beetmove = task.get_upstream_artifacts_with_zip_extract_param(context)
     context.release_props = get_release_props(context)
     context.checksums = dict()  # Needed by downstream calls
     context.raw_balrog_manifest = dict()    # Needed by downstream calls
 
     if context.task['payload'].get('artifactMap'):
-        context.artifacts_to_beetmove = _extract_and_check_maven_artifacts_to_beetmove(
```

Comment:
I'm a little bit worried about this change. If we do so, I think we need to land the geckoview changes on all branches, otherwise, it would work. What do you think?

Reply:
Yes, that's correct. I figured since it's mergeday, we have a good window to land and uplift everywhere since there' some expected freeze/breakage timeline. The alternative is to enrich the existing `if` with checking of the `maven.zip` within the `upstreamArtifacts`. We can do that as well.

Reply:
@JohanLorenzo I added another check to allow us a gradual rollout without the need of landing the in-tree patches. Nor the github ones. 

I tested locally against an [old task](https://tools.taskcluster.net/groups/NQDz9u91QgePXkyLj7Tl9A/tasks/e6fUc7noSBigu9i3xYKZVA/details) and a [new task](https://tools.taskcluster.net/groups/DYmBqSrNTWqb7b6_tTL9xw/tasks/WpX4bA1pTgyISs0Qs93O3A/details)